cmd/livenessprobe/main.go

```diff
@@ -78,11 +79,12 @@ func (h *healthProbe) checkProbe(w http.ResponseWriter, req *http.Request) {
 // acquireConnection wraps the connlib.Connect but adding support to context
 // cancelation.
 func acquireConnection(ctx context.Context) (conn *grpc.ClientConn, err error) {
+	liveMetricsManager := metrics.NewCSIMetricsManager("")
```

Comment:
Do we want to expose args to configure metrics gathering like the other sidecars?

Reply:
IMO,  we dont have to. The main reason being (iic) we haven't got any requests yet to have it exposed.  isnt it?

Reply:
I think it would be good to be able to expose a metrics endpoint so that we're consistent with all the other sidecars. Monitoring tools could potentially detect issues if a driver starts taking too long to respond to the probe.

Reply:
@msau42 ok.. sure. can we keep it separate from this PR ?

Reply:
Sure but I think we should get it into this next release so that we're consistent with the other sidecars

Reply:
@msau42 yes, lets try for that. can you approve this PR

Reply:
This is added to this PR itself :+1: